tf2onnx/onnx_opset/rnn.py

```diff
@@ -10,15 +10,14 @@
 from __future__ import unicode_literals
 
 import logging
-
 import numpy as np
 from tf2onnx import utils
 from tf2onnx.handler import tf_op
 
 logger = logging.getLogger(__name__)
 
 
-# pylint: disable=unused-argument,missing-docstring
+# pylint: disable=unused-argument,missing-docstring,invalid-name
```

Comment:
can you move this behind variable that triggers this, like 'var  # pylint: disable=...' so it does not apply to the full file.